src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -307,8 +323,9 @@ export default {
 			flex-basis: calc(100% / 8);
 			text-align: center;
 
-			&:hover::before {
-				background-color: var(--color-background-hover);
```

Comment:
Should we lose th default hover? I think we should keep it, no?

Reply:
Please not that the default styling is still here. Meaning there is still a rounded background, but it's not compatible with dark theme anymore. We need to put back the `background-color: var(--color-background-hover);`

Reply:
![image](https://user-images.githubusercontent.com/14975046/158791648-2819c918-6aca-427e-86b3-ba664c17c911.png)